panther_manager/config/safety.xml

```diff
@@ -2,45 +2,72 @@
 <root BTCPP_format="4">
   <BehaviorTree ID="Safety">
     <Sequence>
-      <Sequence name="BatteryStatusSequence"
-                _skipIf="battery_health == POWER_SUPPLY_HEALTH_GOOD \
-|| battery_health == POWER_SUPPLY_HEALTH_UNKNOWN">
+      <Sequence name="BatteryOverheatSequence"
+                _skipIf="battery_health != POWER_SUPPLY_HEALTH_OVERHEAT">
+        <Sequence name="FatalBatTempSequence"
+                  _skipIf="bat_temp &lt;= FATAL_BAT_TEMP">
+          <SignalShutdown reason="Battery status unhealthy"/>
+        </Sequence>
+        <Sequence name="CriticalBatTempSequence"
+                  _skipIf="bat_temp &lt;= CRITICAL_BAT_TEMP">
+          <CallTriggerService name="TriggerEStop"
+                              service_name="hardware/e_stop_trigger"
+                              timeout="100"
+                              _skipIf="e_stop_state"/>
```

Comment:
Should e-stop be triggered when critical battery temp is reached or when battery signals overheat?

Reply:
According to the arrangement with Radek, should be triggered when critical battery temp is reached